data/workflow.dot

```diff
@@ -0,0 +1,98 @@
+digraph graphname {
+    newrank = true;
+    size="26,10";
+    rankdir="LR";
+    splines=ortho;
+    graph [fontname = "sans"];
+    node [fontname = "sans", style=filled];
+    edge [fontname = "sans", color=gray50];
+    raw_share[
+        label="Shareable",
+        shape=diamond, color="#C7E8CA", fillcolor="#C7E8CA"
+        ];
+    pre_share[
+        label="Shareable",
+        shape=diamond, color="#C7E8CA", fillcolor="#C7E8CA"
+        ];
+    ana_share[
+        label="Shareable",
+        shape=diamond, color="#C7E8CA", fillcolor="#C7E8CA"
+        ];
+    plo_share[
+        label="Shareable",
+        shape=diamond, color="#C7E8CA", fillcolor="#C7E8CA"
+        ];
+    a[label="Analog  ", shape=cds, color="#BBBBBB", fillcolor=grey93, fontcolor="#999998"];
+    tim[label="Time-domain\nk-space Data", shape=folder, color="#BBBBBB", fillcolor=grey93];
+    vol[label="Volumetric\nReconstruction", shape=folder, color="#BBBBBB", fillcolor=grey93];
+    dic[
+        label="DICOM",
+        shape=folder, color="#BBBBBB", fillcolor=grey93
+        ];
+    heh[
+        label="Heuristic\n(built-in or user-defined)",
+        shape=note, color="#F7D060", fillcolor=grey93
+        ];
+    heu[
+        label="Heudiconv",
+        shape=doubleoctagon, color="#F7D060", fillcolor="#FFF2B2", fontcolor="#F05555"
+        ];
+    raw[
+        label="Standardized Raw Data\n(e.g. BIDS NIfTI)",
+        shape=folder, color="#BBBBBB", fillcolor=grey93
+        ];
+    pre[
```

Comment:
could you only split account for that multistage process -- add that "manual curation" step or a shortcut over it

Those items below -- although on one hand I like mentioning on those extra derivatives, since they have nothing to do  with heudiconv -- probably best to not keep but use space for those conversion stages.  I would may be though mention bids-validator as an example of BIDS-App which could indeed be immediately applied. Then for bids-validator we can write text that it is beneficial to run it as soon as possible after data acquisition to catch problems with actual data acquisition (inconsistent fieldmaps or other parameters across sequences/subjects)

Reply:
> Those items below

I agree that the last line of folders is not heudiconv-specific, but it gives the bigger picture — and makes a visually strong case for it: heudiconv moves you from scanner land to shareable land.
Same goes for the bids validator, right? Heudiconv doesn't seem to use `bidsschematools.validate`.

> could you only split account for that multistage process

Perhaps breaking this up into two figures might be better? This one for the role heudiconv plays in the overall data workflow, and another one for the internal flowchart. Like having a second figure which details how Heudiconv gets from DICOM to “Standardized Raw Data”.

I'm also a bit concerned that the internal flowchart might be a bit overwhelming.
Based on what we walked through there is:
* Heuristic selection
* Heuristic specification
* Form adaptation given via the CLI given a specific heuristic
* Form adaptation based on the .txt generated by a dummy run of the heuristic.

The latter being cyclical, right?
I actually think we might want to sit down or to do a doodle or you could send me one, not sure how to arrange those processes.